smt/surrogate_models/krg_based.py

```diff
@@ -1463,6 +1463,8 @@ def _predict_values(self, x: np.ndarray, is_acting=None) -> np.ndarray:
             X_cont = (X_cont - self.X_offset) / self.X_scale
 
         else:
+            n_eval, n_features_x = x.shape
```

Comment:
`n_feature_x` does not seem to be used below, either replace by `_` or this line could stay before the `if`